src/renderer/store/transcription/reducer.ts

```diff
@@ -61,10 +61,11 @@ const transcriptionReducer: Reducer<Transcription | null, Action<any>> = (
 
     // Have to check this if to get rid of linter error
     if (pasteyWords !== undefined && suffix !== undefined) {
```

Comment:
Please update this to 'pasted words' - is this your spelling mistake? 😄

Reply:
that is left over from @Aretle

Reply:
🚌